api/db/populate.js

```diff
@@ -61,7 +61,7 @@ const populateCollection = async () => {
       break;
     default:
       console.log(
-        `Unknown collection '${userArgs.c}', possible options are: [interests, users, events, attendants, comments, friendships, messages]`
+        `Unknown collection '${userArgs.c}', possible options are: [interests, users, events, attendants, comments, friendships, messages, notifications]`
```

Comment:
I think this should go now, as the notifications collection can't be populated standalone